docs/migrate-to-managed-asm/virtual-service-in-cluster-asm.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START servicemesh_migrate_to_managed_asm_virtual_service_virtualservice_frontend_ingress]
+# This VirtualService splits HTTP traffic coming into the cluster you're migrating away from.
+# This VirtualService ensures that the traffic will be shared between both clusters.
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: frontend
```

Comment:
We cannot re-use [`/docs/migrate-to-managed-asm/virtual-service.yaml`](https://github.com/GoogleCloudPlatform/anthos-service-mesh-samples/blob/nimjay-virtual-service-in-cluster-asm/docs/migrate-to-managed-asm/virtual-service.yaml) because the name has to be different — `name: frontend`, not `name: frontend-ingress`.

Why does the name have to be `frontend`?

Because in the new tutorial, I will be using [`/docs/shared/online-boutique/virtual-service.yaml`](https://github.com/GoogleCloudPlatform/anthos-service-mesh-samples/blob/main/docs/shared/online-boutique/virtual-service.yaml#L18). It's `name: frontend`.